### ISSUE TITLE
Add latest ConstraintLayout version number

### ISSUE THREAD

**Issue by alenz316** [ConstraintLayout/constraintlayout.github.io#69]
The site should list the latest version of ConstraintLayout and have convenience gradle dependency as well.

**Comment by writtmeyer** [comment]
I agree that that would be useful. We could put the information on an extra page (gradle config). 
Mentioning it either in the side menu or on the starter page doesn't cut it, since with ConstraintLayout there generally is a final, older version and a beta or alpha version with new features.
The downside is, that we have to upgrade this information regularly and thus the information will be tremporarily outdated whenever a new CL version is announced.

**Comment by rock3r** [comment]
Agree with @writtmeyer, could be a page in the basics sectio, and maybe just a "latest version" indication on the sidebar.

**Comment by writtmeyer** [comment]
Will take care of it.

**Comment by rock3r** [comment]
Thanks :)

**Comment by danielpassos** [comment]
Should it be closed since the PR is already merged?
